AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -445,20 +456,22 @@ public void onClick(View view) {
             mLastClickTime = SystemClock.elapsedRealtime();
             mTimeoutHandler.removeCallbacks(mShowQuestionTask);
             int id = view.getId();
-            if (id == R.id.flashcard_layout_ease1) {
-                Timber.i("AbstractFlashcardViewer:: EASE_1 pressed");
-                answerCard(Consts.BUTTON_ONE);
-            } else if (id == R.id.flashcard_layout_ease2) {
-                Timber.i("AbstractFlashcardViewer:: EASE_2 pressed");
-                answerCard(Consts.BUTTON_TWO);
-            } else if (id == R.id.flashcard_layout_ease3) {
-                Timber.i("AbstractFlashcardViewer:: EASE_3 pressed");
-                answerCard(Consts.BUTTON_THREE);
-            } else if (id == R.id.flashcard_layout_ease4) {
-                Timber.i("AbstractFlashcardViewer:: EASE_4 pressed");
-                answerCard(Consts.BUTTON_FOUR);
-            } else {
-                mCurrentEase = 0;
+            if(mPrevCard==mCurrentCard) {
```

Comment:
Spacing - and it's best to invert the if to reduce nesting.

```suggestion
            if (mPrevCard == mCurrentCard) {
```